templates/index.html

```diff
@@ -335,13 +335,35 @@ <h4 class="title"></h4>
                 </li>
 
             </ul>
+
+            <div class="sidebar-title">סינון דיונים ותאונות</div>
+            <ul>
+                <li style="height: 30px">
+                    <div class="input-append dropdown">
+                        <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+                            <span class="add-on" style="border: 1px solid #ccc; border-radius: 3px 3px 0 0; margin-left:-1px"><i class="icon-filter"></i></span>
+                            <input id="view-filter" value="הצג את כלל התאונות והדיונים" style="padding-right:7px; border: 1px solid #ccc; height:26px; width:204px; border-radius: 0 0 3px 3px; position:absolute;"/>
```

Comment:
You can just write: הצג תאונות ודיונים



‫ב-1 ביולי 2015, בשעה 06:33, ‏‏Daniel Hershcovich ‏<notifications@github.com> כתב/ה:‬

> In templates/index.html:
> 
> > @@ -335,13 +335,35 @@ <h4 class="title"></h4>
> >                  </li>
> >  
> >              </ul>
> > +
> > +            <div class="sidebar-title">סינון דיונים ותאונות</div>
> > +            <ul>
> > +                <li style="height: 30px">
> > +                    <div class="input-append dropdown">
> > +                        <a class="dropdown-toggle" data-toggle="dropdown" href="#">
> > +                            <span class="add-on" style="border: 1px solid #ccc; border-radius: 3px 3px 0 0; margin-left:-1px"><i class="icon-filter"></i></span>
> > +                            <input id="view-filter" value="הצג את כלל התאונות והדיונים" style="padding-right:7px; border: 1px solid #ccc; height:26px; width:204px; border-radius: 0 0 3px 3px; position:absolute;"/>
> כלל should be כל?
> 
> —
> Reply to this email directly or view it on GitHub.
>